src/js/jsx/shared/SplitButton.jsx

```diff
@@ -34,31 +34,35 @@ define(function (require, exports, module) {
      */
```

Comment:
If you're touching this file, please check all of it's instances to make sure you're not breaking any existing interactions.